InvenTree/templates/js/translated/tables.js

```diff
@@ -1,4 +1,4 @@
-{% load i18n %}
+{% load js_i18n %}
```

Comment:
@SchrodingersGat have you looked into overriding the tag? That should be possible and reduce custom code.